protostar/commands/cairo1_commands/build_cairo1_command.py

```diff
@@ -79,7 +81,12 @@ async def build(
         relative_cairo_path: Optional[list[Path]] = None,
         target_contract_name: str = "",
     ) -> None:
+
         cairo_path = relative_cairo_path or []
+
+        libraries = maybe_fetch_linked_libraries(self._project_root_path)
+        cairo_path += libraries
```

Comment:
I think it should be an error if you simultaneously use Scarb and link libraries in protostar.toml